src/main/java/io/pakland/mdas/githubstats/domain/model/HistoricQueries.java

```diff
@@ -0,0 +1,25 @@
+package io.pakland.mdas.githubstats.domain.model;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "historic_queries")
+public class HistoricQueries {
+    @Id
+    @Column(name = "id", updatable = false, nullable = false)
```

Comment:
```suggestion
    @Column(updatable = false, nullable = false)
```
if the entity attribute name and DB field name are equal, it isn't needed to specify the "name" param on @Column annotation. The same criteria applies to the other class attributes.